### PR TITLE
add random ports in tests

### PR DESCRIPTION
**Pull request by brastak**
For #9 comments

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#10](https://github.com/artipie/npm-proxy-adapter/issues/10) is assigned to @hdouss[/z](https://www.0crat.com/u/hdouss), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/23f78445-b607-42b5-a966-89ee415d8e90); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Review by HDouss — DISMISSED** [review]
@brastak Thanks

**Comment by HDouss** [comment]
@g4s8 Good for me. Please review and/or merge.

File: src/test/java/com/artipie/npm/proxy/NpmProxyITCase.java [hunk 1]
```diff
@@ -96,6 +97,14 @@
      */
     private VertxSliceServer srv;
 
+    static {
+        try (ServerSocket socket = new ServerSocket(0)) {
```
Comment (g4s8) [inline src/test/java/com/artipie/npm/proxy/NpmProxyITCase.java]: @brastak can we move it to `@BeforeAll` or `@BeforeEach` setup method?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@brastak just one comment

File: src/test/java/com/artipie/npm/proxy/NpmProxyITCase.java (hunk 1 above)
Comment (brastak) [reply src/test/java/com/artipie/npm/proxy/NpmProxyITCase.java]: done. moved to beforeall

**Comment by HDouss** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/npm-proxy-adapter/issues/10#issuecomment-618594127))

@HDouss This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#10](https://github.com/artipie/npm-proxy-adapter/issues/10) is [in scope](http://www.zerocracy.com/policy.html#14) for 2days
  * The role is `REV`
  * The job is assigned to @hdouss[/z](https://www.0crat.com/u/hdouss) for [2days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @hdouss[/z](https://www.0crat.com/u/hdouss) will get 25% of their hourly rate for this job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @brastak[/z](https://www.0crat.com/u/brastak): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fnpm-proxy-adapter%2310%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by HDouss** [comment]
@g4s8 how to trigger rewards here?

**Comment by g4s8** [comment]
@0crat out

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @hdouss[/z](https://www.0crat.com/u/hdouss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by sereshqua** [comment]
@brastak even though quality is already bad here, please be aware that next you would need to start all you comments with the name of the users they are referred to, [see](https://github.com/artipie/npm-proxy-adapter/pull/10#discussion_r413023915)

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by brastak** [comment]
@sereshqua 

> @brastak even though quality is already bad here, please be aware that next you would need to start all you comments with the name of the users they are referred to, [see](https://github.com/artipie/npm-proxy-adapter/pull/10#discussion_r413023915)

@sereshqua Why? I have been requested for changes by @g4s8 . I fixed it, closed conversation and and re-requested review. GitHub should to do all machinery itself

**Comment by sereshqua** [comment]
@brastak - core reviewer has found 0 issues

> The quality of a job is “bad” if any of the following happens (these violations we can’t forgive):
> 
> The code reviewer didn’t find any issues in the source code changes of a pull request and just accepted it.

**Comment by HDouss** [comment]
@brastak quality is bad for the job performer who is me in this review task. Nothing related to you.

**Pull request merged**